app/src/main/java/com/github/fribourgsdp/radio/Playlist.kt

```diff
@@ -3,7 +3,8 @@ package com.github.fribourgsdp.radio
 import kotlinx.serialization.Serializable
 
 @Serializable
-data class Playlist (var name: String, var genre: Genre){
+data class Playlist (var name: String, var genre: Genre): java.io.Serializable{
```

Comment:
do you really need to implement java.io.Serializable instead of directly using the Json parser like I have done in the user saving and loading? I think we should choose a single parsing library and use it

Reply:
(this is still an import, just with kotlin syntax instead of java syntax)